### PR TITLE
Leave halcs-fe services running when AFC BPM board is removed

### PR DESCRIPTION
**Pull request by guilhermerc**

### CODE REVIEW

**Comment by guilhermerc** [comment]
@ericonr , can we merge this?

File: src/apps/halcsd/init/systemd/etc/udev/rules.d/80-udev_fpga.rules [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 KERNEL=="fpga*", ACTION=="add", OWNER="root", GROUP="users", MODE="660", PROGRAM="/usr/local/share/halcs/scripts/generate-dev-idx.sh %k", SYMLINK+="fpga/%c{1}", RUN+="/usr/bin/systemctl --no-block start halcs-ioc@%c{2}.target halcs-ioc@%c{3}.target"
```
Comment (ericonr) [inline src/apps/halcsd/init/systemd/etc/udev/rules.d/80-udev_fpga.rules]: I'd first make a commit removing this file, it's not used by the build system anymore.

**Review by ericonr — COMMENTED** [review]
For the commit message, I'd use some word other than "brick"; IMO that implies it's stuck forever in a bad state, instead of just having to be restarted. Probably something like "locks up" ?

File: src/apps/halcsd/init/systemd/etc/udev/rules.d/80-udev_fpga.rules (hunk 1 above)
Comment (guilhermerc) [reply src/apps/halcsd/init/systemd/etc/udev/rules.d/80-udev_fpga.rules]: I've fixed and properly parameterized the build system (`80-udev_fpga.rules` was actually being used by it) so it's not installed anymore.

**Review by ericonr — APPROVED** [review]

**Pull request merged**